fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1251,20 +1250,9 @@ public void testGetObjectGraphWithChildren() throws IOException {
         httpGet.setHeader("Limit", Integer.toString(CHILDREN_LIMIT));
         try (final CloseableHttpResponse response = execute(httpGet)) {
             try (final CloseableGraphStore graph = getGraphStore(response)) {
-                assertTrue("Should contain child count!",
-                        graph.contains(ANY, createURI(location), HAS_CHILD_COUNT.asNode(), ANY));
-
-                final Iterator<Quad> children = graph.find(ANY, createURI(location), HAS_CHILD_COUNT.asNode(), ANY);
-                assertTrue("Should have a child count triple!", children.hasNext());
-
-                // Total child count should be provided
-                final Node child = children.next().getObject();
-                assertTrue("Object should be a literal!", child.isLiteral());
-                assertEquals(CHILDREN_TOTAL, Integer.parseInt(child.getLiteralValue().toString()));
-
                 final Iterator<Quad> contains = graph.find(ANY, createURI(location), CONTAINS.asNode(), ANY);
                 assertTrue("Should find contained child!", contains.hasNext());
-                assertEquals(CHILDREN_LIMIT - 1, Iterators.size(contains));
+                assertEquals(CHILDREN_LIMIT, Iterators.size(contains));
```

Comment:
The "Limit" header determines the number of triples that are returned from the `ChildrenRdfContext` RDF generation... not specifically restricted to the `ldp:contains` triples.
Previously, the number of `ldp:contains` triples that were returned was `limit - 1` because there used to be an additional triple that was included in the response: `fedora:numberOfChildren`.

Now that the `fedora:numberOfChildren` has been removed, the only triples that `ChildrenRdfContext` generates are `ldp:contains` triples; therefore, the "Limit" header exactly corresponds to the number of `ldp:contains` triples returned to the caller.